src/commands/install-composer.yml

```diff
@@ -0,0 +1,15 @@
+description: Install Composer
+
+parameters:
+  install-flags:
+    type: string
+    default:  --install-dir=/usr/local/bin --filename=composer
+    description: |
+      By default, composer will be installed at "/usr/local/bin/composer", use this to override the standard install flags.
```

Comment:
It's probably worth renaming this to `install-dir` and removing the filename flag, it's unlikely we'll need to get super granular with the install flags and filename.

Reply:
done in https://github.com/CircleCI-Public/php-orb/pull/25/commits/52c92358746202130f4165e331d2e40fcd1a0df1

Note: Hardcoding `--filename=composer` because https://getcomposer.org/installer defaults to install composer as `composer.phar`